sbt-plugin-test/build.sbt

```diff
@@ -105,10 +116,10 @@ lazy val multiTest = crossProject.
       consoleWriter +: (jsExecutionFiles in Test).value
     },
 
-    // Test crossPlatform (as a setting, it's evaluated when loading the build)
-    crossPlatform ~= { value =>
-      assert(value == JSPlatform,
-          "crossPlatform should be JSPlatform in multiTestJS")
+    // Test platformDepsCrossVersion (as a setting, it's evaluated when loading the build)
+    platformDepsCrossVersion ~= { value =>
+      assert(value eq ScalaJSCrossVersion.binary,
+          "platformDepsCrossVersion should be ScalaJSCrossVersion.binary in multiTestJS")
```

Comment:
I think that test can be removed, no?

Reply:
No, it is the only test that makes sure that [this line](https://github.com/scala-js/scala-js/pull/3197/files#diff-dd593403434ec9de3670cb5722f10171R452) has the intended effect (and does not go away).

The tests of sbt-platform-deps, in turn, ensure that if `platformDepsCrossVersion` is correctly set up, then `%%%` works as expected. But it is the responsibility of `ScalaJSPlugin` to set up `platformDepsCrossVersion`, and therefore we need to test that.